src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -115,24 +107,40 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
       amdIn <- getAmdXml(datasetId, amdFile)
       amdOut <- configuration.userTransformer.transform(amdIn)
       maybeProvenance = provenance.collectChangesInXmls(Map(
-      "http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/" -> compare(amdIn, amdOut),
-      "http://easy.dans.knaw.nl/schemas/md/ddm/" -> compare(oldDcmi, newDcmi),
+        "http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/" -> compare(amdIn, amdOut),
+        "http://easy.dans.knaw.nl/schemas/md/ddm/" -> compare(oldDcmi, newDcmi),
       ))
+      shaToPath = sha1Manifest(bag.getPayLoadManifests).asScala
+        .map { case (path, sha) => sha -> bagDir.relativize(File(path)) }.toMap
+      doi = depositProps.getString("identifier.doi")
+      _ = trace(bagInfo, doi)
+      migratedFiles <- configuration.preStagedProvider.get(doi) // paths from migration info
+      migratedPayloadFiles = migratedFiles.filterNot(_.path.toString.startsWith("easy-migration/")) // exclude metadata migrated as data for provenance
```

Comment:
Should we really assume these files are not in the bag? It is true for a bag freshly exported from fedora, but might not be true for a bag that has been converted and is somehow fixed with another run. Rerunning would require some manual corrections of the deposit: at least restoring the user account in bag-info.txt and removing the deposit.properties file.